bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md

```diff
@@ -0,0 +1,1150 @@
+
+Sjors Provoost: 00:04:17
+
+Yeah, I'd say so.
+I think there is a release, attempted release schedule on the Bitcoin Core repository, which I think is for mid-October.
+
+Aaron van Wirdum: 00:04:28
+
+So, there are a bunch of new features as well as bug fixes and improvements, sorry, performance improvements in this new release.
+We're going to discuss a couple of these.
+This is kind of just our selection.
+If this podcast was made by two other people, by Ors and Sharon, they might make a different selection.
+But we, Sjors and Aaron, we made the selection.
+
+Sjors Provoost: 00:04:59
+
+Yeah, I mean, you can look at all the commits, so all the atomic changes that go into a release, and there could be a thousand of them, but most of them are very boring, or at least they are very boring unless you are really into details of how compilers work or into cleaning up code for the sake of cleaning it up.
+So there's a lot of changes that are just not very interesting to discuss, although they are important and they should be, they are being, you know, they always get reviewed because any of those changes could of course be a really scary thing.
+
+Aaron van Wirdum: 00:05:29
+
+Yeah, I'm just saying we made a selection of some of the changes that we think are probably the most interesting
+to you, our dear listener.
+
+## Peer-to-Peer network changes
+
+Sjors Provoost: 00:05:38
+
+That's right.
+
+Aaron van Wirdum: 00:05:40
+
+Okay, so let's start.
+Sjors, I think that this one is the first one mentioned in the release notes as well.
+It's about the peer-to-peer and network changes.
+Is that a typo?
+Should you just say peer-to-peer changes?
+Peer-to-peer network changes?
+Whatever.
+
+Sjors Provoost: 00:05:55
+
+Peer-to-peer networking.
+
+Aaron van Wirdum: 00:05:57
+
+Let's move on.
+So it says that there's a change and you're gonna explain to me what this is That's the format of the podcast in case you forgot.
+It says download headers from peers so the downloading from Downloading of headers from peers has been reworked.
+That's right somehow.
+
+## Download headers from peers
+
+Sjors Provoost: 00:06:18
+
+Yeah, this one might be worth a separate episode, but I think we can explain the gist of it now.
+So when your node first starts up, it is going to find peers, which we discussed in an earlier episode, and then it's going to download headers from peers, which we've also discussed in earlier episodes.
+So it tries to get all the headers first.
+
+Aaron van Wirdum: 00:06:36
+
+Hang on, so every block has a header which is essentially the hash of the block itself, right?
+
+Sjors Provoost: 00:06:41
+
+Plus a little bit of extra info that, for example, contains the timestamp and the amount of proof of work.
+Well, the amount of proof of work follows in the hash.
+
+Aaron van Wirdum: 00:06:52
+
+It's basically hash plus some extra info.
+Yes.
```

Comment:
This "Yes" was misattributed to Aaron. There are instances where phrases like 'Right' or 'Yeah' from one speaker mistakenly appear in the text attributed to another speaker. Please remove these occurrences for clarity and accuracy.